media/js/firefox/challenge-the-default/challenge-the-default.es6.js

```diff
@@ -79,10 +88,20 @@ kittenButton.addEventListener(
                 kittenButton.focus();
             }
         });
+
+        // UA
         window.dataLayer.push({
             event: 'in-page-interaction',
             eAction: 'Kitten modal'
         });
+
+        // GA4
+        window.dataLayer.push({
+            event: 'widget_action',
+            type: 'easter egg',
+            action: 'discover',
+            name: 'kitten modal'
+        });
```

Comment:
`type: 'easter egg'` and `action: 'discover'` feels a bit too one-off maybe? 

I wonder if this one could be a more generic modal type event? Not sure what to do with the others though :confused: